client/components/styles/globalStyle.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from 'styled-components';
+
```

Comment:
Nice. Good way to global style. maybe you could incorporate the color changing there.